debian/confluent-ksqldb.spec.in

```diff
@@ -11,8 +11,7 @@ Vendor: Confluent, Inc.
 Packager: Confluent Packaging <packages@confluent.io>
 BuildArch: noarch
 
-Requires: confluent-rest-utils
-
+Provides: community-ksqldb = 0.14.0
```

Comment:
previously, the release process didn't include bumping anything in these files - at a minimum, we should add this as a manual step to the release wiki but that is of course always error prone (as people tend to miss manual steps). Is there any way to either automate this or have an automatic verification that we're not producing duplicate packages? (Or if this is in your plans and this is just something incremental, that is OK as well)

Reply:
> Is there any way to either automate this or have an automatic verification that we're not producing duplicate packages?

I'll do some Makefile hackery so this will be an input into the process to define the value to make it optional.

Reply:
sounds good, feel free to have that as a follow-up or just tacked onto this PR (I assume you're talking about the Makefile in this folder) otherwise LGTM

Reply:
I spent yesterday attempting to tune things here, but unfortunately Debian doesn't make this very easy to have "inputs" to their package metadata (debian/control). Any solution I come up with will be rather ugly here, so for the time being I think we need to march forward with this change (I added 1 more enhancement), as the change still "improves" the code base.

We can address your concerns in the release job.